core/controllers/base.py

```diff
@@ -504,7 +522,14 @@ def init_csrf_secret(cls):
 
     @classmethod
     def _create_token(cls, user_id, issued_on):
-        """Creates a digest (string representation) of a token."""
+        """Creates a digest (string representation) of a token.
+        Args:
+          user_id: User Id.
+          issued_on: Round time to seconds.
+
+        Returns:
+          token: Digest in Base64.
```

Comment:
Indentation is wrong; return value should not have a name.

Description could be clearer:

    Returns:
        str. The generated CSRF token.